### PR TITLE
Updated the overlay text of the screenshot button

### PR DESCRIPTION
**Pull request by Mammoets**
Updated the overlay text of the screenshot button to be more descriptive.

### Description

When a test is completed, one of the buttons on the page is the screenshot button. The current overlay text for the screenshot button says "Save screenshot", which is not adequately descriptive as the button copies the screenshot to the clipboard rather than saving it. This might have caused confusion or slowed down the user interaction process. The updated text, "Copy screenshot to clipboard," more accurately describes the button's functionality, making it clearer and more intuitive for the users.

### Closes 

The updated text, "Copy screenshot to clipboard," more accurately describes the button's functionality. There are no issues related to this PR.

### CODE REVIEW

File: frontend/src/ts/commandline/lists/result-screen.ts [hunk 1]
```diff
@@ -117,7 +117,7 @@ const commands: MonkeyTypes.Command[] = [
   },
   {
     id: "saveScreenshot",
-    display: "Save screenshot",
+    display: "Copy screenshot to clipboard",
     icon: "fa-image",
     alias: "ss picture",
```
Comment (Miodec) [inline frontend/src/ts/commandline/lists/result-screen.ts:122]: Add `save` as alias

File: frontend/static/challenges/sourcecode.txt [hunk 2]
```diff
@@ -5352,7 +5352,7 @@ backend/anticheat
                   </div>
                   <div
                     id="saveScreenshotButton"
-                    aria-label="Save screenshot"
+                    aria-label="Copy screenshot to clipboard"
```
Comment (Miodec) [inline frontend/static/challenges/sourcecode.txt:5355]: Revert this change please.

**Review by Miodec — CHANGES_REQUESTED** [review]

**Comment by Mammoets** [comment]
Fixed comments

**Comment by Mammoets** [comment]
fixed comments

**Comment by Miodec** [comment]
Thanks

**Pull request merged**